polynote-frontend/polynote/ui/layout/splitview.ts

```diff
@@ -116,7 +118,15 @@ export class SplitView extends Disposable {
             });
         });
 
-        this.el = div(['split-view'], [left, leftDragger, center, rightDragger, right]);
+        const initialClasses = [];
+        if (initialPrefs.leftPane.collapsed) {
+            initialClasses.push('left-collapsed');
+        }
+        if (initialPrefs.rightPane.collapsed) {
+            initialClasses.push('right-collapsed');
+        }
+
+        this.el = div(['split-view', ...initialClasses], [left, leftDragger, center, rightDragger, right]);
```

Comment:
I think this change is no longer necessary (looks like we both fixed the same bug) 😄